### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by stefdoerr**

### CODE REVIEW

File: htmd/queues/localqueue.py [hunk 1]
```diff
@@ -115,8 +115,10 @@ def _createJobScript(self, fname, workdir, runsh, gpudevice=None):
             # Move completed trajectories
             if self.datadir is not None:
                 datadir = os.path.abspath(self.datadir)
-                if not os.path.isdir(datadir):
+                try:
                     os.mkdir(datadir)
+                except OSError:
```
Comment (raimis) [inline htmd/queues/localqueue.py]: Do you try to catch the case that the directory already exists?

Comment (stefdoerr) [reply htmd/queues/localqueue.py]: yes

Comment (stefdoerr) [reply htmd/queues/localqueue.py]: There is a weird race condition when creating the directory on /tmp/ that Alberto hit upon.

Comment (stefdoerr) [reply htmd/queues/localqueue.py]: Well maybe OSError is too broad. I could catch FileExists error

**Comment by stefdoerr** [comment]
Hm now I see how fork PRs work. This PR will track my master until I close it. Maybe I should make branches and PRs then......

File: htmd/queues/localqueue.py (hunk 1 above)
Comment (raimis) [reply htmd/queues/localqueue.py]: Or just use 
```python
os.makedirs(datadir, exist_ok=True)
```
without any exception catching...

File: htmd/queues/localqueue.py [hunk 2]
```diff
@@ -117,7 +117,7 @@ def _createJobScript(self, fname, workdir, runsh, gpudevice=None):
                 datadir = os.path.abspath(self.datadir)
                 try:
                     os.mkdir(datadir)
-                except OSError:
+                except FileExistsError:
                     pass
```
Comment (raimis) [inline htmd/queues/localqueue.py]: Or just use 
```python
os.makedirs(datadir, exist_ok=True)
```
without any exception catching...

Comment (stefdoerr) [reply htmd/queues/localqueue.py]: Ah yes. I sometimes forget that one exists... Thanks

**Pull request merged**